src/SprykerEco/Yves/Computop/Mapper/Init/AbstractMapper.php

```diff
@@ -75,11 +81,11 @@ public function __construct(
     }
 
     /**
-     * @param \Spryker\Shared\Kernel\Transfer\TransferInterface|\Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
      *
-     * @return \Generated\Shared\Transfer\ComputopCreditCardPaymentTransfer|\Generated\Shared\Transfer\ComputopPayPalPaymentTransfer
+     * @return \Spryker\Shared\Kernel\Transfer\TransferInterface
```

Comment:
This is a bad idea.
first of all - Sniffer will complain about this.

Reply:
I know what are you talking about, as you said previously the best solution would be some kind of abstract transfer object. I'll add this later if I have time.